components/form-check/examples/30-disabled-state.js

```diff
@@ -37,7 +37,7 @@ function Example({ brand }) {
 				</Option>
 			</FormCheck>
 
-			<FormCheck type="radio" name="example-radio-medium-disabled" defaultValue="2">
```

Comment:
We added those default values to see what the select state looks like from a design perspective. If that's not a11y we should not advertise it but not remove it from the tests

Reply:
@dominikwilkowski I had to remove the defaultValue in these instances because it was set to an option that was disabled which made the entire radio-group untabbable to making it impossible to reach to a keyboard user.